tests/testsrc/test262.properties

```diff
@@ -5253,7 +5254,7 @@ language/expressions/new 41/59 (69.49%)
 
 ~language/expressions/new.target
 
-language/expressions/object 865/1169 (73.99%)
+language/expressions/object 863/1169 (73.82%)
```

Comment:
Any idea what's going on here? The stats for updated, but I don't see any tests being unbroken in this section

Reply:
Not sure, could a previous PR have fixed something but forgot to run spotlessApply?

Reply:
Well, the last thing that got merged was another one of your PRs 🤔

Reply:
Ok no idea then 😅

Reply:
I have the impression i have seen something similar for other pr's also, but i might be wrong

Reply:
You probably need to rebase this branch. `language/expressions/object 863/1169 (73.82%)` is the value currently in the master branch. It's not being added with this PR.

Reply:
Will do, thanks